packages/gatsby-mdx/context.js

```diff
@@ -68,13 +68,9 @@ exports.withMDXScope = withMDXScope;
 var MDXScopeProvider = (exports.MDXScopeProvider = function MDXScopeProvider(
   _ref2
 ) {
-  var __mdxScope = _ref2.__mdxScope,
```

Comment:
Updated `context.js` to take in multiple scopes instead of combining them all into one.